scripts/jenkins/red_hat/red_hat_docker_certification.py

```diff
@@ -0,0 +1,89 @@
+import argparse
+import sys
+import time
+
+import requests
+
+
+def has_scan_errors(digest, pid, api_key):
```

Comment:
The script is Python 2.7 compatible, verified locally. Unsure which Python is in the release image.